### DISCUSSION TITLE
Bad sound quality after RPi Zero flashed with new image 202101311

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#159] [Testing] [answered]
@rern 

As requested I have now flashed a new SD card with the latest RPi Zero image (which gives Kernel 5.10.11-3-ARCH armv6l) and update 202101311. All seems to go OK and I was able to restore a backup of my previous setting (by placing backup.gz in /boot, which is something I have never done before).  All the relevant settings seem to have been restored as I would expect. 

As you know I use this Zero with a pair of HK Soundstick speakers via bluetooth so there is no other soundcard/connection other than the bluetooth connection. Unfortunately when I started to play albums or web radios the sound, which had been very good with the previous versions, sounded strange.  The best way to describe it would be like an old fashioned steam locomotive struggling with a heavy load.  The sound was all there but broken up with, perhaps, tiny gaps in the stream so it sounded slloowww!

 I hope this is something that can be fixed.

Chris

**Comment by rern** [comment]
Just downloaded, flashed and updated to 202101311 with no audio devices attached.
I'm listening to Radio Paradise - Mellow Mix with my Bluetooth headphones right now without any issues. Local files are playing just fine as well.

**Comment by ceejayemm** [comment]
@rern 
Redownloaded latest PI Zero image (5.10.11-3-ARCH armv6l) and flashed to a new SD card, updated to 202101311. Booted PI and configured for scratch (no backup restore), Sound still poor on all sources (webradio and NAS based albums) to the point where it is not worth listening for more than a few minutes. Same installation was perfect on previous image. Tried different Sound Profiles, no difference.

Chris

**Reply by rern** [reply]
An updated kernel, 5.10.11-4, was released. Try `pacman -Syu` then reboot to see if it's any better.

**Reply by ceejayemm** [reply]
@rern 
Makes no discernable difference I am afraid.

Chris

**Comment by ceejayemm** [comment, marked as answer]
Fixed it.

It turns out that there is known interference (on my RPi Zero at least) between the onboard wireless network adapter and Bluetooth adapter which I discovered after lots of Googling. 

The solution in my case was to disable the onboard Bluetooth setup by adding 'dtoverlay=disable-bt' to /boot/config.txt. I then added a USB based Bluetooth adapter and rebooted. On  reboot the speakers had to be reattached but once done the sound quality was back to normal.

Chris
